arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -1877,6 +1877,22 @@
         "properties": {
           "type": "object",
           "description": "Gets or sets the resource properties."
```

Comment:
Yes, we have a task to remove gets or sets across all our swagger files. For this change, I'll use the old autorest that won't add the duplicated gets/sets